src/ensembl/src/shared/components/layout/StandardAppLayout.scss

```diff
@@ -0,0 +1,162 @@
+@import 'src/styles/common';
+
+$sideBarContentWidth: 320px;
+$sideBarToolStripWidth: 46px;
+$topBarHeight: 40px;
+$drawerWindowWidth: 45px;
+
+.standardAppLayout {
+  height: 100%;
+  overflow: hidden;
+}
+
+.topBar {
+  display: grid;
+  grid-template-columns: 1fr $sideBarContentWidth;
+  grid-column-gap: $sideBarToolStripWidth;
+  align-items: center;
+  height: $topBarHeight;
+  background: $light-grey;
+  box-shadow: 0 2px 3px $grey;
+  position: relative;
+  padding-left: 18px;
+  z-index: 200;
+}
+
+.mainWrapper {
+  height: calc(100% - #{$topBarHeight});
+  position: relative;
+}
+
+.main {
+  height: 100%;
+  transition: margin 0.3s ease-in-out;
+
+  &Default {
+    margin-right: calc(#{$sideBarContentWidth} + #{$sideBarToolStripWidth});
+  }
+
+  &FullWidth {
+    margin-right: $sideBarToolStripWidth;
+  }
+}
+
+.sideBarWrapper {
+  position: absolute;
+  top: 0;
+  left: 0;
```

Comment:
Do we need this `left: 0` here?

Reply:
Not as such, no; I added it just for clarity, because I wasn't sure of the default value of the left property in an absolutely positioned element, and wanted to be explicit. I can remove it if you want.

Reply:
I am quite confused why this is not placing the sidebar completely to the left?

Reply:
It is :-)
Then the whole sidebar-cum-drawer block it transform-translated by 100% minus the width of the sidebar to the right

Reply:
Right! I get it now.